### PR TITLE
Recommended Workflow for IDR: Enrich with VirusTotal

### PR DESCRIPTION
**Pull request by sengleson-r7**
New workflow that we will add to the Discover page to recommend to IDR customers.

Reviewers: If you have the means, please try to import and test this workflow from an IDR investigation!

### CODE REVIEW

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal.icon [hunk 1]
```diff
@@ -0,0 +1,2800 @@
...
+  "kom": {
+    "komandVersion": "1.60.2-0-g80196",
+    "komFileVersion": "2.0.0",
+    "exportedAt": "2020-08-31T23:38:27.510036662Z",
+    "workflowVersions": [
+      {
+        "name": "Enrich InsightIDR Alerts with Threat Intelligence from VirusTotal",
+        "type": "runnable",
+        "version": "",
+        "description": "This workflow parses indicators (IPs, URLs, domains, and/or hashes) from an InsightIDR alert and adds threat intelligence from VirusTotal. This workflow requires a VirusTotal API key, which can be generated for free by creating an account on VirusTotal's website (https://virustotal.com)",
```
Comment (mrinehart-r7) [inline workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal.icon:11]:
```suggestion
        "description": "This workflow parses indicators (IP addresses, URLs, domains, and/or hashes) from an InsightIDR alert and adds threat intelligence from VirusTotal. This workflow requires a VirusTotal API key, which can be generated for free by creating an account on VirusTotal's website (https://virustotal.com)",
```

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# Requirements
+
+* InsightIDR License
+* InsightConnect License
+
+# Documentation
+
+## Setup
+
+**This workflow requires an Insight Orchestrator and an API Key from VirusTotal!**
```
Comment (mrinehart-r7) [inline workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:23]:
```suggestion
**This workflow requires an InsightConnect Orchestrator and an API Key from VirusTotal!**
```

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+If you have not yet deployed and activated an Orchestrator, then you'll need to either deploy the OVA or run the install script on a vanilla server. Start by visiting our Help docs here: https://docs.rapid7.com/insightconnect/install-and-activate-the-orchestrator
+
+Once you have activated on Orchestrator, you'll need an API key from VirusTotal.
```
Comment (mrinehart-r7) [inline workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:27]:
```suggestion
Once you have activated an Orchestrator, you'll need an API key from VirusTotal.
```

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Description
```
Comment (mrinehart-r7) [inline workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:1]: Consider making the links in here clickable w/ markdown syntax

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md (hunk 2 above)
Comment (sengleson-r7) [reply workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:23]: We actually just call it the Insight Orchestrator! 
https://docs.rapid7.com/insightconnect/install-and-activate-the-orchestrator/#install-and-activate-the-insight-orchestrator

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md (hunk 4 above)
Comment (sengleson-r7) [reply workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:1]: good call, updated!

File: workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md (hunk 2 above)
Comment (mrinehart-r7) [reply workflows/Enrich_InsightIDR_Alerts_With_Threat_Intelligence_From_VirusTotal/help.md:23]: Today I learned! Thanks!

**Pull request merged**